content/posts/configuration/sections/education/index.es.md

```diff
@@ -56,6 +56,11 @@ degrees:
     url: "#"
   - title: Nullam vitae orci tincidunt purus viverra pulvinar.
     url: "#"
+  custonSections: #(opcional)
+    - name: Tesis
```

Comment:
Look like a typo here.

Reply:
In spanish it's written like this. Thank you for checking it anyway!

Reply:
Ah. Thank you for explaining.